src/utils/scripts/id_rsa_pub.pem

```diff
@@ -0,0 +1,13 @@
+-----BEGIN RSA PUBLIC KEY-----
```

Comment:
This file will also need to be removed

Reply:
🤔 ... i can still see the file here

Reply:
cries.. i wanna do it while u see.. to make sure im doing it right next time.